apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.csproj

```diff
@@ -38,12 +38,6 @@
     <Compile Update="SpannerCommand.cs">
       <SubType>Component</SubType>
     </Compile>
-    <Compile Update="SpannerConnection.cs">
```

Comment:
odd -- I did not remove them, but the files still are there.  Maybe they are dupe entries?